### ISSUE TITLE
Change the reply whitespace (topic flag)

### ISSUE THREAD

**Issue by silentrob** [silentrob/superscript#198]
I'm proposing a new topic flag that changes how the whitespace behaves on multi-line replies.
For example:

```
+ test 
- test reply\n
^ foo\n
^ bar
```

```
> topic:ws topic_name
+ test 
- test reply
^ foo
^ bar
< topic
```

So the following would be identical. "ws" could be named anything.

**Comment by mariusursache** [comment]
What would be the use of this? I think it encourages code inconsistencies… I think that ideally there should be only one way to address multiple lines.

**Comment by kirsle** [comment]
It looks similar to RiveScript's `! local concat = newline`, which applies to the rest of the file after that line, until changed again, but its influence ends when the file ends (so other files aren't affected when loading a whole directory).

**Comment by mariusursache** [comment]
Yes, of course, but I guess it’s saner to do a search and replace in a document to maintain consistency instead of allowing two different syntaxes, and then adding new functionality/complexity to make sure both work.

Btw, which reminds me of a previous discussion: using another parser than RegEx, which would allow better catching various characters and filtering out things. @silentrob thoughts on this?
